src/roswire/proxy/roslaunch/roslaunch.py

```diff
@@ -184,8 +184,8 @@ def launch(self,
             logger.debug(f'instrumenting launch config: {launch_config}')
 
             if launch_prefixes:
-                m = "individual launch prefixes are not yet implemented"
-                raise NotImplementedError(m)
+                launch_config = \
+                    launch_config.with_launch_prefixes(launch_prefixes)
```

Comment:
This is cool, but the use case I'm envisioning will be inefficient with this. Because I want to get the list of the nodes (e.g., all python nodes) automatically, I have to create the `LaunchConfig` beforehand. And then here, we read the launch file again. Maybe it would make sense to allow passing a `LaunchConfig` to this method. What do you think?

Reply:
I see. That makes sense. We could either add a `launch_from_config` method, or change the `filename` parameter to `filename_or_config`. I think the former makes more sense here.